test/acceptance/features/immutabeSuccessfulBinding.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Successful Service Binding Resource should be Immutable 
+
+    As a user of Service Binding operator
+    I should not be able to apply changes of an already appled and successful SBR
```

Comment:
typo: appled

what is SBR?

Reply:
fixed typo. SBR is Service Binding Resource

Reply:
ok, then please avoid the abbreviation for improved clarity.